modules/react/tokens/lib/depth.ts

```diff
@@ -77,8 +77,39 @@ export type CanvasDepth = {
    * ```
    */
   4: {
-    border: '1px solid rgba(218, 226, 230, 1)';
-    boxShadow: '0px 8px 16px 0 rgba(0, 0, 0, 0.12)';
+    boxShadow: '0px 4px 16px rgba(0, 0, 0, 0.2), 0px 8px 32px rgba(0, 0, 0, 0.16)';
+  };
+  /**
+   * ### Depth 5
+   * Used for Popups, Toast Messages, Dialogs, Side Panels (without opacity overlay behaviour)
+   *
+   * @example
+   * ```ts
+   * import { depth } from '@workday/canvas-kit-react/tokens';
+   *
+   * const cardStyles = {
+   *   ...depth[5],
+   * };
+   * ```
+   */
+  5: {
+    boxShadow: '0px 5px 20px rgba(0, 0, 0, 0.2), 0px 10px 40px rgba(0, 0, 0, 0.16)';
+  };
+  /**
+   * ### Depth 6
+   * Used for Modal Dialogs, Side Panels (both with opacity overlay behaviour)
```

Comment:
Q: I'm not sure I understand the "Side Panels (both with opacity overlay behavior)." I'm also not sure why Side Panel is listed in level 5 and 6 when we only have a Side Panel with 6? Can you help me understand that?

Reply:
This is a difference between both depth for Side Panel. CK Side Panel component doesn't have opacity overlay behavior, so it has depth 5. So depth 6 is more like recommended usage for Side Panel with overlay.

<img width="1666" alt="Screen Shot 2022-03-25 at 10 18 15 AM" src="https://user-images.githubusercontent.com/48605821/160160056-3efcde5b-a518-4bbd-afea-25d76e2362eb.png">

Reply:
Cool. Thanks for the explanation!